google-cloud-pubsub/lib/google/cloud/pubsub/service.rb

```diff
@@ -477,6 +477,14 @@ def a_time? obj
           true
         end
 
+        def dead_letter_policy options
+          return nil unless options[:dead_letter_topic_name]
+          Google::Cloud::PubSub::V1::DeadLetterPolicy.new(
+            dead_letter_topic:     options[:dead_letter_topic_name],
+            max_delivery_attempts: options[:dead_letter_max_delivery_attempts]
```

Comment:
What is this is not passed? Since its an optional parameter.

Reply:
I'm sorry, I don't understand the question. Can you maybe describe how you think this should work?

Reply:
Sorry there was a typo. I meant to say:

What if this (`dead_letter_max_delivery_attempts`) is not passed? Since its an optional parameter.

Reply:
Fixed, `max_delivery_attempts` is now only set on the request if `dead_letter_max_delivery_attempts` is passed.